backend/src/common/server-application/constants/api.constants.ts

```diff
@@ -2,8 +2,26 @@ import { AuthApiPath } from '~/bundles/auth/enums/enums.js';
 import { ApiPath } from '~/common/enums/enums.js';
 
 const WHITE_ROUTES = [
-    `/api/v1${ApiPath.AUTH}${AuthApiPath.SIGN_UP}`,
-    `/api/v1${ApiPath.AUTH}${AuthApiPath.SIGN_IN}`,
+    {
+        routerPath: `/api/v1${ApiPath.AUTH}${AuthApiPath.SIGN_UP}`,
+        methods: ['POST'],
+    },
+    {
+        routerPath: `/api/v1${ApiPath.AUTH}${AuthApiPath.SIGN_IN}`,
+        methods: ['POST'],
+    },
+    {
+        routerPath: `/v1${ApiPath.DOCUMENTATION}/`,
+        methods: ['GET'],
+    },
+    {
+        routerPath: `/v1${ApiPath.DOCUMENTATION}/static/*`,
+        methods: ['GET'],
+    },
+    {
+        routerPath: `/v1${ApiPath.DOCUMENTATION}/json`,
+        methods: ['GET'],
+    },
```

Comment:
if you want to use RegExp it should be here. 

in this case, you can combine these routes

```suggestion
    {
        path: new RegExp(`^/api/v1${APIPath.DOCUMENTATION}/*`),
        methods: ['GET'],
    },
```